phoenix-core/pom.xml

```diff
@@ -212,30 +335,16 @@
       <groupId>org.apache.tephra</groupId>
       <artifactId>tephra-core</artifactId>
     </dependency>
-    <dependency>
-      <groupId>org.apache.tephra</groupId>
-      <artifactId>tephra-core</artifactId>
-      <type>test-jar</type>
-      <scope>test</scope>
-    </dependency>
     <dependency>
       <groupId>org.apache.tephra</groupId>
       <artifactId>tephra-hbase-compat-2.0</artifactId>
     </dependency>
 
-    <!-- Make sure we have all the antlr dependencies -->
+    <!-- Other dependencies -->
     <dependency>
       <groupId>org.antlr</groupId>
       <artifactId>antlr-runtime</artifactId>
     </dependency>
-    <dependency>
-      <groupId>jline</groupId>
-      <artifactId>jline</artifactId>
-    </dependency>
-    <dependency>
-      <groupId>sqlline</groupId>
-      <artifactId>sqlline</artifactId>
```

Comment:
Dropping this here, removes sqlline from phoenix-client jar (via phoenix-core).

Either need to keep it here with an exclusion, or add sqlline as a dependency on phoenix-client.

Reply:
Right, I forgot because the code does not refer to it.
I feel that it belongs to client. I also updated it to 1.9 now.